view/src/pages/home.js

```diff
@@ -0,0 +1,199 @@
+    width: drawerWidth,
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+  },
+  content: {
+    flexGrow: 1,
+    padding: theme.spacing(3),
+  },
+  avatar: {
+    height: 110,
+    width: 100,
+    flexShrink: 0,
+    flexGrow: 0,
+    marginTop: 20,
+  },
+  uiProgess: {
+    position: "fixed",
+    zIndex: "1000",
+    height: "31px",
+    width: "31px",
+    left: "50%",
+    top: "35%",
+  },
+  toolbar: theme.mixins.toolbar,
+});
+
+class Home extends Component {
+  state = {
+    render: false,
+  };
+
+  loadAccountPage = (event) => {
+    this.setState({ render: true });
+  };
+
+  loadTodoPage = (event) => {
+    this.setState({ render: false });
+  };
+
+  logoutHandler = (event) => {
+    localStorage.removeItem("AuthToken");
+    this.props.history.push("/login");
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      firstName: "",
+      lastName: "",
+      profilePicture: "",
+      uiLoading: true,
+      imageLoading: false,
+    };
+  }
+
+  componentDidMount = () => {
+    authMiddleWare(this.props.history);
+    const authToken = localStorage.getItem("AuthToken");
+    axios.defaults.headers.common = { Authorization: `${authToken}` };
+    axios
+      .get("/user")
+      .then((response) => {
+        console.log(response.data);
+        this.setState({
+          firstName: response.data.userCredentials.firstName,
+          lastName: response.data.userCredentials.lastName,
+          email: response.data.userCredentials.email,
+          phoneNumber: response.data.userCredentials.phoneNumber,
+          country: response.data.userCredentials.country,
+          username: response.data.userCredentials.username,
+          uiLoading: false,
+          profilePicture: response.data.userCredentials.imageUrl,
+        });
+      })
+      .catch((error) => {
+        if (error.response.status === 403) {
+          this.props.history.push("/login");
+        }
+        console.log(error);
+        this.setState({ errorMsg: "Error in retrieving the data" });
+      });
+  };
+
+  render() {
+    const { classes } = this.props;
+    if (this.state.uiLoading === true) {
+      return (
+        <div className={classes.root}>
+          {this.state.uiLoading && (
+            <CircularProgress size={150} className={classes.uiProgess} />
+          )}
+        </div>
+      );
+    } else {
+      return (
+        <div className={classes.root}>
+          <CssBaseline />
+          <AppBar position="fixed" className={classes.appBar}>
+            <Toolbar>
+              <Typography variant="h6" noWrap>
+                TodoApp
+              </Typography>
+            </Toolbar>
+          </AppBar>
+          <Drawer
+            className={classes.drawer}
+            variant="permanent"
+            classes={{
+              paper: classes.drawerPaper,
+            }}
+          >
+            <div className={classes.toolbar} />
+            <Divider />
+            <center>
```

Comment:
is this an html way of centering? 

(This is why I wasn't a big fan of react...I like to keep my styles separate from my html. Cleaner to read that way, and styles really shouldn't be integral to the structure of the page. But if this is the React way to do it, then by all means)

Reply:
Yeah, the `<center>` tag has been deprecated since HTML5.  I would recommend at least `<div style="text-align: center">` instead.